can/python/bt_plugin_can.py

```diff
@@ -2,6 +2,13 @@
 import cantools
 import struct
 
+
+# It seems that in version 2.0.0, even though the framework supports creation of streams
+# without packet support, the utils.trimmer filter is not capable of parsing such streams.
```

Comment:
Thanks for noticing this, I don't think it's the intention here, I'll investigate.

Reply:
Can you try to remove this and test with the tip of the stable-2.0 branch of babeltrace?  If it works, then we can avoid creating packets for nothing here.  I'll ask when we plan to release babeltrace 2.0.2, which would include this fix.